### ISSUE TITLE
A suggestion (and a question) for Rustual Boy...

### ISSUE THREAD

**Issue by John-Enigma** [yupferris/vb-rs#26]
Can this emulator have the option to play in a Gameboy-like resolution (or view)? 

What do I mean by this? This emulator is about the Virtual Boy, right? A handheld console that had anaglyph 3D. My suggestion would be that if you can actually "fuse" the 3D colors (red and cyan) to make it look like you're playing a Gameboy game. This would actually be perfect for the people that don't have anaglyph glasses, like me.

Also, will this emulator come with its own debugger, and perhaps other tools, for hacking games and creating homebrew games?

**Comment by yupferris** [comment]
> Can this emulator have the option to play in a Gameboy-like resolution (or view)?

Yes, multiple view options are definitely possible, though they're not top-priority right now. See [this ticket](https://github.com/emu-rs/rustual-boy/issues/22) for what we're focusing on most at the moment. After the 1.0 release, "polish" features like this one would make a lot of sense :)

> Also, will this emulator come with its own debugger, and perhaps other tools, for hacking games and creating homebrew games?

It already comes with a basic CLI debugger, although it's not currently documented. This has mostly been useful to track down and fix emulation bugs, but should be quite useful for homebrew/hacking as well (although it's a bit unpolished for now).

**Comment by John-Enigma** [comment]
^Thanks for the reply! Looking forward for the initial 1.0 version of Rustual Boy.

**Comment by yupferris** [comment]
Happy to hear it! :)
